web/modules/custom/dbcdk_community_moderation/dbcdk_community_moderation.module

```diff
@@ -8,9 +8,17 @@
  * Implements hook_user_login().
  */
 function dbcdk_community_moderation_user_login(\Drupal\user\Entity\User $account) {
-  // Redirect moderators to the flagged content list.
-  if (in_array('admin_moderator', $account->getRoles())) {
+  // Redirect users with specific focus to their primary page.
+  // Moderators go to the flagged content page.
+  if ($account->getRoles(TRUE) == ['admin_moderator']) {
```

Comment:
Hmm. I think it should be moved to the "base" module if the same conditional should handle possible roles that are both moderation and editor roles. - Regarding the importances of roles then I believe we should "just" create some by ourselves.

1. Moderation
2. Librarian
3. Editor

I believe this order would cater most to DBC's business case (as many people moderating and reviewing as possible) and then content creation as the last priority since (I assume) this action is the one that will happen the least and will be done by people who understand the system and can easier navigate to their wished location.